vision/body_analysis/arcface/README.md

```diff
@@ -11,7 +11,7 @@ The model LResNet100E-IR is an ArcFace model that uses ResNet100 as a backend wi
 
 |Model        |Download  |Checksum| Download (with sample test data)|ONNX version|Opset version|LFW * accuracy (%)|CFP-FF * accuracy (%)|CFP-FP * accuracy (%)|AgeDB-30 * accuracy (%)|
 |-------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|
-|LResNet100E-IR|    [248.9 MB](https://s3.amazonaws.com/onnx-model-zoo/arcface/resnet100/resnet100.onnx)|[MD5](https://s3.amazonaws.com/onnx-model-zoo/arcface/resnet100/resnet100-md5.txt)|  [226.6 MB](https://s3.amazonaws.com/onnx-model-zoo/arcface/resnet100/resnet100.tar.gz) | 1.3  |8|99.77     | 99.83  |  94.21     | 97.87|
+|LResNet100E-IR|    [248.9 MB](model/arcfaceresnet100-opset8.onnx)|[MD5](model/arcfaceresnet100-opset8-md5.txt)|  [226.6 MB](model/arcfaceresnet100-opset8.tar.gz) | 1.3  |8|99.77     | 99.83  |  94.21     | 97.87|
```

Comment:
These links will likely not work without direct links -- https://github.com/onnx/models/blob/master/vision/body_analysis/arcface/model/arcfaceresnet100-8.onnx

Reply:
Additionally, model file names should be arcfaceresnet100-8.tar.gz and arcfaceresnet100-8.onnx for standardization.

Reply:
its better to not use a direct link because otherwise changes wont work across branches.

Reply:
Alright! While working on a previous PR, I discovered that they didn't work as intended due to "blob" in the URL -- but perhaps that was a testing mistake.